exercises/07_title/main_test.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import "testing"
+
+func TestTitle(t *testing.T) {
```

Comment:
Please be careful with the tests. At least one of them is always given in the problem definition:
![image](https://user-images.githubusercontent.com/65840/147660563-c076569f-55e9-4132-bddb-d7af871d8568.png)